src/components/Calculator.js

```diff
@@ -1,46 +1,115 @@
 import './Calculator.css';
 
-import React from 'react';
+import React, { useState } from 'react';
 import CalculatorScreen from './Calculator/ClaculatorScreen';
 import CalculatorBtn from './Calculator/CalculatorBtn';
 import calculate from '../modules/logic/calculate';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+function Calculator() {
```

Comment:
- [ ] Good job so far by refactoring your code :clap:  But since React uses ES6, I think you should use some of the new features like Arrow Functions for the reason that it is a good practice and also this can make for example a lot easier to predict functions behavior when passed as callbacks, and prevents bugs caused by use of this within callbacks. :smile:  Please do it :smile: